### PR TITLE
Lesson_5_Protractor

### PR DESCRIPTION
**Pull request by ErnestYedigaryan**

### CODE REVIEW

File: Lesson_5_Protractor/spec.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    beforeEach(function () {
+        browser.get('https://angular.io/');
+    });
+
+    browser.manage().timeouts().implicitlyWait(20000);
+
+    it('should have a result', () => {
+        searchBar.sendKeys('What is Angular');
+
+        expect(searchResults.isDisplayed());
```
Comment (LauraQA) [inline Lesson_5_Protractor/spec.js:25]: Ernest jan don't you need any kind of wait here?

File: Lesson_5_Protractor/spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        expect(searchResults.isDisplayed()); 
+        
+        ngDoc.click();
+
+        expect(ngDocHeader.isDisplayed());
+        expect(ngDocArguments.isDisplayed());
+        expect(ngDocOptions.isDisplayed());
+
+        // browser.sleep(5000);
+
```
Comment (LauraQA) [inline Lesson_5_Protractor/spec.js]: Please remove all the unnecessary comments.

File: Lesson_5_Protractor/conf.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+exports.config = {
+    framework: 'jasmine',
+    seleniumAddress: 'http://localhost:4444/wd/hub',
+    specs: ['spec.js']
+};
```
Comment (LauraQA) [inline Lesson_5_Protractor/conf.js:9]: Ernest jan where are the browser configurations? ))

**Review by LauraQA — COMMENTED** [review]
Ernest jan there is a missed task and a few comments that need to be done. But it's good with changes will be better

File: Lesson_5_Protractor/spec.js (hunk 1 above)
Comment (ErnestYedigaryan) [reply Lesson_5_Protractor/spec.js:25]: Laura jan when I type browser.manage().timeouts().implicitlyWait(20000);, it waits for all locators to appear.

File: Lesson_5_Protractor/conf.js (hunk 3 above)
Comment (ErnestYedigaryan) [reply Lesson_5_Protractor/conf.js:9]: fixed

File: Lesson_5_Protractor/spec.js (hunk 2 above)
Comment (ErnestYedigaryan) [reply Lesson_5_Protractor/spec.js]: fixed

**Pull request merged**